app/src/Components/Base/Content.js

```diff
@@ -8,14 +8,51 @@ import DataSource from './../DataSource/DataSource';
 import NotFoundComponent from '../Error/NotFoundComponent'
 import Login from '../Login/Login';
 
+const PrivateRoute = ({ component: Component, ...rest }) => (
+    <Route {...rest} render={(props) => (
+        isAuthentificated(rest)
+            ? <Component {...props} />
+            : <Redirect to={{
+            pathname: '/login',
+            state: { from: props.location }
+        }} />
+    )} />
+);
+
+function isAuthentificated(props){
+    let cookieValue = getCookieValue('token');
```

Comment:
That was just WIP, isn't there any better way to get a cookie?

Reply:
no